LICENSE

```diff
@@ -1,9 +1,29 @@
-Copyright (c) 2021, Center for Advanced Systems Understanding (CASUS)
+BSD 3-Clause License
 
+Copyright (c) 2021, atoMEC
```

Comment:
I would switch this and give the copyright to CASUS?

Copyright (c) 2021, Center for Advanced Systems Understanding (CASUS)